src/main/java/nl/topicus/mssql2monetdb/util/MssqlUtil.java

```diff
@@ -36,8 +36,15 @@ public static boolean allMSSQLTablesHaveData(Map<String, CopyTable> tablesToCopy
 
 			if (rowCount == 0)
 			{
-				LOG.error(table.getFromName() + " is empty! Stopping all copy actions!");
-				return false;
+				if (table.isAllowEmpty())
+				{
+					LOG.warn("{} is empty", table.getDescription());
```

Comment:
het gedeelte '{} is empty' met de tabel beschrijving is hetzelfde in de else, dus misschien handiger om dit in een variable te zetten?